Resources/Maps/_Goobstation/meta.yml

```
@@ -118574,6 +118574,13 @@
     - type: Transform
       pos: -17.38306,-3.5393329
       parent: 5350
+- proto: MaterialSilo
+  entities:
+  - uid: 26992
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)